src/lib/y2storage/device.rb

```diff
@@ -193,6 +197,19 @@ def hash
     #       sorted list (less than)
     storage_class_forward :compare_by_name
 
+    # Check whether the device exists in the probed devicegraph
+    #
+    # @note The probed devicegraph is the sanitized version of the
+    #   initial raw probed.
+    #
+    # @see StorageManager#raw_probed
+    # @see exists_in_devicegraph?
+    #
+    # @return [Boolean]
```

Comment:
to bring some light into bootloader usage of this method. It is used mainly to detect which partition is "new", so not exists in probed graph. It is used to determine which partition create specifically for booting purpose like new PREP partition and in such case bootloader prefer this newly created partition. So for bootloader, I think it does not matter much if it is sanitized version or not.

Reply:
Ok, so in yast-bootloader we can keep the code as it is now, but do you prefer to use raw version there?

Reply:
well, I do not care. As long as method allow me to identify if partition is newly created ( so not exists in probed ).